Description: Remove unavailable features
 As we use Debian packaged libzstd not the bundled, some features are
 not (can not be) enabled at build time and unavailable, or made no-op:
 - debug, fat-lto, no_asm, thin, thin-lto: only useful with bundled zstd
 - zstdmt: deprecated, upstream hidden by default
 - legacy: Debian package has it on
 - pkg-config: always used
 - std: no-op with use-std-c-ffi.patch
Last-Update: 2023-11-27
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,3 +63,2 @@
 [features]
-debug = []
 default = [
@@ -69,5 +68,3 @@
 experimental = []
-fat-lto = []
 legacy = []
-no_asm = []
 no_wasm_shim = []
@@ -76,5 +73,2 @@
 std = []
-thin = []
-thin-lto = []
 zdict_builder = []
-zstdmt = []
